executor/test_test.go

```diff
@@ -15,6 +15,16 @@ func TestCopyin(t *testing.T) {
 	}
 }
 
+func TestCsumInet(t *testing.T) {
+	switch res := testCsumInet(); {
```

Comment:
add a helper function along the lines of:
```
func testWrapper(t *testing.T, f func() int) {
	switch res := f(); {
	case res < 0:
		t.Skip()
	case res > 0:
		t.Fail()
	default:
	}
}
```

Then all tests can be reduced to:
```
func TestCsumInet(t *testing.T) {
	testWrapper(t, testCsumInet)
}
```